contracts/PolygonTokenBridger.sol

```diff
@@ -40,14 +40,36 @@ contract PolygonTokenBridger is Lockable {
     // WETH contract on Ethereum.
     WETH9 public immutable l1Weth;
 
+    // Chain id for the L1 that this contract is deployed on or communicates with.
+    // For example: if this contract were meant to facilitate transfers from polygon to mainnet, this value would be
+    // the mainnet chainId 1.
+    uint256 public immutable l1ChainId;
+
+    // Chain id for the L2 that this contract is deployed on or communicates with.
+    // For example: if this contract were meant to facilitate transfers from polygon to mainnet, this value would be
+    // the polygon chainId 137.
+    uint256 public immutable l2ChainId;
+
+    modifier onlyChainId(uint256 chainId) {
```

Comment:
nit instead of using modifier, does calling internal method _requireChainId directly in `send` and `retrieve` use less gas? I think its readable either way

Reply:
My understanding is that modifiers are usually inlined, so I expect that it would be the same either way.